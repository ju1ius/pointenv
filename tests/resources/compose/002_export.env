export EXP_1="exported" EXP_2='exported 2'

  export EXP_3="whitespace at start"
