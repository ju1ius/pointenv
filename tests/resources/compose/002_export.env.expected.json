[
  {
    "key": "EXP_1",
    "value": "exported"
  },
  {
    "key": "EXP_2",
    "value": "exported 2"
  },
  {
    "key": "EXP_3",
    "value": "whitespace at start"
  }
]
